tests/conformance/scripts/run-tests/index.ts

```diff
@@ -69,9 +73,11 @@ async function main(options: {
           hadError = true;
           continue;
         }
-        if (output !== expectedOutput) {
+        const outputNorm = normalizeJson(output);
+        const expectedNorm = normalizeJson(expectedOutput);
```

Comment:
expected should already be normalized, right?

Reply:
Yeah although maybe we should leave this here in case we decide to rebuild the output from another runner than ts? Maybe it's not needed.